tests/jest/system/dverbose-flag.spec.ts

```diff
@@ -27,9 +27,12 @@ test('inspect on dverbose-project pom using -Dverbose', async () => {
     'expected-dverbose-dep-graph.json',
   );
   const expectedDepGraph = depGraphLib.createFromJSON(expectedJSON).toJSON();
-  result = result.scannedProjects[0].depGraph?.toJSON();
+  const result = res.scannedProjects[0].depGraph?.toJSON();
 
   expect(result).toEqual(expectedDepGraph);
+  expect(
```

Comment:
relevant test for the maven plugin version

Reply:
could be brittle depending on the version of maven used during test - better to test on a project that defines the maven-dependency-plugin version?